hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -90,7 +90,7 @@ expiry.throttleResource=expiry-throttle.json
 contracts.redirectTokenCalls=true
 contracts.referenceSlotLifetime=31536000
 contracts.scheduleThrottleMaxGasLimit=5000000
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```

Comment:
I think we should add to `hedera-services/hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties` for default values

Reply:
oh nvmd , its correct in that file